client/src/app/pages/source-platforms/components/platform-detail-drawer.tsx

```diff
@@ -0,0 +1,125 @@
+  DescriptionListDescription,
+  Tabs,
+  Tab,
+  TabTitleText,
+} from "@patternfly/react-core";
+import spacing from "@patternfly/react-styles/css/utilities/Spacing/spacing";
+import { SourcePlatform } from "@app/api/models";
+
+import { PageDrawerContent } from "@app/components/PageDrawerContext";
+import PlatformAppsTable from "./platform-applications-table";
+import { SchemaDefinedField } from "@app/components/schema-defined-fields/SchemaDefinedFields";
+
+export interface IPlatformDetailDrawerProps {
+  onCloseClick: () => void;
+  platform: SourcePlatform | null;
+}
+
+enum TabKey {
+  Details = 0,
+  Coordinates,
+  Applications,
+}
+
+const PlatformDetailDrawer: React.FC<IPlatformDetailDrawerProps> = ({
+  onCloseClick,
+  platform,
+}) => {
+  const { t } = useTranslation();
+
+  const [activeTabKey, setActiveTabKey] = React.useState<TabKey>(
+    TabKey.Details
+  );
+
+  return (
+    <PageDrawerContent
+      isExpanded={!!platform}
+      onCloseClick={onCloseClick}
+      focusKey={platform?.id}
+      pageKey="platform-details"
+      header={
+        <TextContent>
+          <Text component="small" className={spacing.mb_0}>
+            {t("titles.platformDrawer")}
+          </Text>
+          <Title headingLevel="h2" size="lg" className={spacing.mtXs}>
+            {platform?.name}
+          </Title>
+        </TextContent>
+      }
+    >
+      <div>
+        <Tabs
+          activeKey={activeTabKey}
+          onSelect={(_event, tabKey) => setActiveTabKey(tabKey as TabKey)}
+        >
+          <Tab
+            eventKey={TabKey.Details}
+            title={<TabTitleText>{t("terms.details")}</TabTitleText>}
+          >
+            <DescriptionList className="platform-detail-drawer-list">
+              <DescriptionListGroup>
+                <DescriptionListTerm>{t("terms.name")}</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {platform?.name}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  {t("terms.providerType")}
+                </DescriptionListTerm>
+                <DescriptionListDescription>
+                  {platform?.kind}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>{t("terms.url")}</DescriptionListTerm>
+                <DescriptionListDescription>
+                  {platform?.url}
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+            </DescriptionList>
+          </Tab>
+          <Tab
+            eventKey={TabKey.Coordinates}
+            title={<TabTitleText>{t("terms.coordinates")}</TabTitleText>}
+          >
+            <DescriptionList className="platform-coordinates-drawer-list">
+              <SchemaDefinedField />
+            </DescriptionList>
```

Comment:
Does the `DescriptionList` wrapper need to be there?  It makes sense in the Details tab, but not so much in the Coordinates or Applications tabs.